### PR TITLE
Feature/python-fy and unify decompile script

### PR DESCRIPTION
**Pull request by B13rg**
This batch of changes converts the decompile process from a sh script to a full python script.
Instead of configuration happening in the shell scripts, it can be done with cmd line parameters or env vars.

It works by reading values from ENV vars (like the script) and calling `os.system()` for external commands.

This also removes the `decompile.sh` script and replaces it with the python function.
Functionality is the same

### CODE REVIEW

File: src/parser/decompiler/decompile.py
Comment (hassayag) [inline src/parser/decompiler/decompile.py:1]: While we are here, I think the `decompiler` fits better a folder above (`src/decompiler`), as it is its own independent process that should live at the same level as the parser and bot code

**Review by hassayag — CHANGES_REQUESTED** [review]
Bash script was getting out of hand, good call. Added a couple missing dotenv calls which I needed to get mine working.

File: src/parser/decompiler/decompile.py
Comment (B13rg) [reply src/parser/decompiler/decompile.py:1]: Moved it to it's own directory like you suggested.
Also moved the `kv3_to_json.py` script to the `utils` directory

File: src/parser/decompiler/localization.py [hunk 1]
```diff
@@ -28,3 +23,10 @@
                 output_file_json = os.path.join(output_folder, filename.replace('.txt', '.json'))
                 with open(output_file_json, 'w', encoding='utf-8') as f:
                     json.dump(out, f, indent=4)
+
+if __name__ == '__main__':
+    # Arg1: Input folder which contains all localization files
+    # Arg2: Output folder which all localization files will be placed in
+    input_folder = sys.argv[1]
+    output_folder = sys.argv[2]
+    process_localization_files(input_folder, output_folder)
```
[the remark this answers is not in the document]
Comment (B13rg) [reply src/parser/decompiler/localization.py]: Removed!  Moved all cli stuff to `deadbot.py`

File: src/parser/decompiler/kv3_to_json.py [hunk 2]
```diff
@@ -51,10 +51,12 @@ def remove_subclass(path):
...
+    kv3_to_json(data, out_path)
 
-if __name__ == '__main__':
-    remove_subclass(sys.argv[1])
 
-    data = kv3.read(sys.argv[1])
-    out_path = sys.argv[2]
-    kv3_to_json(data, out_path)
+if __name__ == '__main__':
+    process_file(sys.argv[1], sys.argv[2])
```
[the remark this answers is not in the document]
Comment (B13rg) [reply src/parser/decompiler/kv3_to_json.py]: Done!

**Comment by B13rg** [comment]
Will have to figure out merge timing with #41 and #49 , I might just wait until after those because so much moves around

**Comment by B13rg** [comment]
Going to break this up into multiple PRs

**Comment by B13rg** [comment]
Okay!
First pass is in https://github.com/deadlock-wiki/deadbot/pull/51

**Pull request closed without merging**